code/modules/shuttle/navigation_computer.dm

```diff
@@ -1,7 +1,7 @@
 /obj/machinery/computer/camera_advanced/shuttle_docker
 	name = "navigation computer"
 	desc = "Used to designate a precise transit location for a spacecraft."
-	z_lock = ZLEVEL_STATION_PRIMARY
+	z_lock = list(ZLEVEL_STATION_PRIMARY)
```

Comment:
@Cyberboss 

What is the sanest way to make this use GLOB.z_levels_station

Late initialize and then update the list?

Reply:
i think there's a hidden function that's run called init()

so you could do, i think

```
z_lock = GLOB.z_levels_station
```

Reply:
no, just set it in Initialize()

Reply:
Ditto